kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyDeployment.java

```diff
@@ -120,6 +122,16 @@ private Container proxyContainer(Context<KafkaProxy> context) {
         // @formatter:off
         var containerBuilder = new ContainerBuilder()
                 .withName("proxy")
+                .withNewLivenessProbe()
+                .withNewHttpGet()
+                .withPath("/healthz")
+                .withPort(new IntOrString(METRICS_PORT_NAME))
```

Comment:
should our concept become `management` port, combining metrics/health etc?

Reply:
Yes I think we should call this "management" everywhere

Reply:
I've adopted the naming at the port level already.  I raised #1903 to capture the need to refactor the config model (again).

Reply:
I feel like we've gone full circle on this ♻️ but yes we should use `management` everywhere.